src/it/resources/com/google/checkstyle/test/chapter4formatting/rule451wheretobreak/InputMethodParamPad.java

```diff
@@ -5,82 +5,9 @@
 
 class InputMethodParamPad
 {
-    void test()
-    {
-        int x = 1 +
-                 2 -
-            3
```

Comment:
Removed unnecessary code from input file. The `MethodParamPad` & `OperatorWrap` both modules had almost same input file. So in this input file I have kept the code only related to `MethodParamPad`.